src/components/ui/pagination/pagination.tsx

```diff
@@ -83,7 +84,8 @@ export const Pagination = ({
       <StyledLink
         onClick={onPreviousClick}
         $disabled={currentPage === 1}
-        aria-label="Next Page"
+        aria-label="Previous Page"
+        tabIndex={currentPage === 1 ? -1 : 0}
```

Comment:
Thanks for the catch it's really bad to have a button that goes nowhere. Your fix has a good idea: take it out of the tab list to prevent it from being accessed, but it's wrong as it won't remove it from the a11y tree. The navigator will still list it as an element (I used the Voice Over rotator and the list of links to check it there you can see Previous Page being listed).
The link should be disabled. This will remove it entirely from the a11y tree and the tab index will be removed automatically.

![CleanShot 2024-01-16 at 17 37 36](https://github.com/satellytes/satellytes.com/assets/1701755/aee92c47-6050-45a8-ab07-222433e5a8f9)